<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>temp_conv_ui</class>
 <widget class="QWidget" name="temp_conv_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>295</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="clearButton">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="enterButton">
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>45</y>
     <width>141</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="leftHand_Char">
      <property name="text">
       <string>Char</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="leftHand_Dec">
      <property name="text">
       <string>Decimal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="leftHand_Hex">
      <property name="text">
       <string>Hexadecimal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>45</y>
     <width>141</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="rightHand_Char">
      <property name="text">
       <string>Char</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rightHand_Dec">
      <property name="text">
       <string>Decimal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rightHand_Kelvin">
      <property name="text">
       <string>Hexadecimal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Output">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>225</y>
     <width>141</width>
     <height>36</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Input">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>225</y>
     <width>141</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert From</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>190</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert To</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Output</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Input</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
